testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/MixedDeploymentOverlayTestCase.java

```diff
@@ -33,7 +33,6 @@
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.SERVER_GROUP;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.STEPS;
 import static org.jboss.as.controller.descriptions.ModelDescriptionConstants.UNDEPLOY;
-import static org.jboss.as.domain.management.ModelDescriptionConstants.TEST;
```

Comment:
I guess this is left over from the split relying on a constant "test" from core!